### PR TITLE
Annotate init files

### PR DESCRIPTION
**Pull request by Kludex**
Related to #1511

### CODE REVIEW

File: kombu/transport/__init__.py [hunk 1]
```diff
@@ -58,7 +61,7 @@ def resolve_transport(transport=None):
         except KeyError:
             if '.' not in transport and ':' not in transport:
                 from kombu.utils.text import fmatch_best
-                alt = fmatch_best(transport, TRANSPORT_ALIASES)
+                alt = fmatch_best(transport, list(TRANSPORT_ALIASES.keys()))
```
Comment (auvipy) [inline kombu/transport/__init__.py:64]: why  list(TRANSPORT_ALIASES.keys()) ?

Comment (Kludex) [reply kombu/transport/__init__.py:64]: It was my mistake!

**Comment by Kludex** [comment]
The `fmatch_best` was actually wrong annotated. Thanks for the review!

**Review by auvipy — APPROVED** [review]

**Comment by Kludex** [comment]
Thanks @auvipy ! 🙇

**Pull request merged**